src/qibocal/auto/task.py

```diff
@@ -108,14 +106,14 @@ def run(self, folder: Path, platform: Platform, qubits: Qubits) -> Results:
         path = self.datapath(folder)
         if operation.platform_dependent and operation.qubits_dependent:
             if platform is not None:
-                if self.action.qubits:
-                    qubits = allocate_qubits(platform, self.action.qubits)
-            if qubits:
+                if len(self.qubits) != 0:
+                    qubits = allocate_qubits(platform, self.qubits)
+            if len(qubits) != 0:
```

Comment:
```suggestion
                if len(self.qubits) > 0:
                    qubits = allocate_qubits(platform, self.qubits)
            if len(qubits) > 0:
```

As specific as possible :)

Reply:
Already fixed :)